src/main/java/com/yugi/application/web/UserController.java

```diff
@@ -25,6 +25,12 @@ public User getUser(@PathVariable("id") long id) {
         return user;
     }
 
+    @RequestMapping(method = RequestMethod.DELETE, value="/user/{id}")
+    public Boolean removeUser(@PathVariable("id") long id) {
+        User user = mapper.removeUser(id);
+        return true;
```

Comment:
**Suggestion:** The removeUser method in the controller is returning a hardcoded true value. It should return a meaningful value based on the operation's success.
```suggestion
    @RequestMapping(method = RequestMethod.DELETE, value="/user/{id}")
    public ResponseEntity<?> removeUser(@PathVariable("id") long id) {
        try {
            mapper.removeUser(id);
            return new ResponseEntity<>(HttpStatus.OK);
        } catch (Exception e) {
            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
        }
    }
```